### PR TITLE
feat(pci.projects.new): add a banner to help create incident ticket

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | DTRSD-79698
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
1. Add new banner for create project on step1 and step2 to help customers to create support ticket

## Related
- [x] TRANS-42899

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/new/config/index.html [hunk 1]
```diff
@@ -71,4 +71,28 @@ <h1 data-translate="pci_project_new_config_title"></h1>
...
+            data-translate="pci_project_new_config_create_project_info"
+        ></span>
+        <a
+            class="d-block"
+            data-ng-href="{{:: $ctrl.supportTicketLink}}"
+            target="_blank"
+            rel="noopener"
+        >
+            <span data-translate="pci_project_new_config_create_project_info_link"></span>
+            <span aria-hidden="true"></span>
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/new/config/index.html:95]: Are you displaying some icon here or is there another reason to use `aria-hidden` here?

Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/new/config/index.html:95]: Ah no, i have to remove it, we had one but we removed it.
Forgot to delete the accessibility related span.

Thanks.

**Review by vks-ovh — DISMISSED** [review]

**Review by JeremyDec — DISMISSED** [review]

**Review by ghyenne — DISMISSED** [review]

**Review by JeremyDec — DISMISSED** [review]

File: packages/manager/modules/pci/src/projects/new/components/support-ticket-banner/component.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import controller from './controller';
+import template from './template.html';
+
+const component = {
+  bindings: {},
```
[the remark this answers is not in the document]
Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/projects/new/components/support-ticket-banner/component.js:5]: I let it to help for adding new variables easily. 
there is an issue ?

[the remark this answers is not in the document]
Comment (JeremyDec) [reply packages/manager/modules/pci/src/projects/new/components/support-ticket-banner/component.js]: It is more in the way "Don't do more than what is needed" :)

**Review by JeremyDec — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Pull request merged**